bin/install

```diff
@@ -31,8 +31,15 @@ install_deno() {
 
     case "$(uname -m)" in
       x86_64) architecture="x86_64" ;;
+      arm64) architecture="aarch64" ;;
       *) fail "Unsupported architecture" ;;
     esac
+    
+    if [ $architecture = "aarch64" ]; then
+      if [[ $version < "1.6.0" ]] || [ $platform != "apple-darwin" ]; then
+        fail "Unsupported architecture"
+      fi
+    fi
```

Comment:
Basically, we don't plan to implement this kind of version-specific support.

Reply:
Ok, the version check is only there to prevent attempting to download a non-existent release since aarch64 was only added recently. Would you rather do nothing there and just let it fail at the end in that situation?

Reply:
That's enough.